src/dotenv_vault/__init__.py

```diff
@@ -1 +1 @@
-from .main import load_dotenv
+from .main import load_dotenv, load_dotenv_vault
```

Comment:
This shouldn't be exposed to the developer as a public method.

https://github.com/motdotla/dotenv/blob/master/lib/main.js#L167

usage for developer should not ever change. just `load_dotenv` and it is smart enough to determine to load .env.vault or .env depending if DOTENV_KEY is set or not.

Reply:
👍  i removed load_dotenvv_ault

Reply:
looks like it is still there?? in `__int__.py` ?

Reply:
Oops! Forgot to push 😎